packages/manager/modules/bm-server-components/src/general-information/install/image/components/config-drive/controller.js

```diff
@@ -0,0 +1,52 @@
+import some from 'lodash/some';
+
+import { DOCUMENTATION_LINK, SUPPORTED_SSH_KEY_FORMATS } from './constants';
+
+export default class DedicatedServerInstallImageConfigDriveCtrl {
+  /* @ngInject */
+  constructor(atInternet) {
+    this.atInternet = atInternet;
+    this.documentationLink = DOCUMENTATION_LINK;
+    this.supportedSshFormatsNames = SUPPORTED_SSH_KEY_FORMATS.map(
+      ({ name }) => name,
+    );
+
+    this.validateSshFormat =
+      DedicatedServerInstallImageConfigDriveCtrl.validateSshFormat;
+  }
+
+  /*= =====================================
+  =            Initialization            =
+  ====================================== */
+
+  $onInit() {
+    this.addMetadata();
+  }
+
+  /* -----  End of Initialization  ------*/
+
+  addMetadata() {
+    this.model.configdrive.metadata.push({
+      added: false,
+      model: {},
+    });
+  }
+
+  static validateSshFormat(sshKey) {
+    return some(SUPPORTED_SSH_KEY_FORMATS, ({ regex }) => regex.test(sshKey));
```

Comment:
SUPPORTED_SSH_KEY_FORMATS.some(({ regex }) => regex.test(sshKey))

Reply:
it's applied